examples/ai/package.json

```diff
@@ -28,6 +28,7 @@
     "ai": "^4.3.16",
     "immer": "^10.1.1",
     "lucide-react": "^0.474.0",
+    "ollama-ai-provider-v2": "^0.0.5",
```

Comment:
Ollama-ai-provider is not actively maintained, switch to ollama-ai-provider-v2	https://github.com/vercel/ai/issues/4700